packages/ChangesReporting/Output/ConsoleOutputFormatter.php

```diff
@@ -101,11 +101,15 @@ private function reportFileDiffs(array $fileDiffs): void
     }
 
     /**
-     * @param SystemError[] $errors
+     * @param array<SystemError|string> $errors
      */
     private function reportErrors(array $errors): void
     {
         foreach ($errors as $error) {
+            if (! $error instanceof SystemError) {
+                $this->outputStyle->error($error);
```

Comment:
is it means that the error will still shown? what message error that will be shown?

Reply:
This is for support the error message is string. If the error must be an object https://github.com/rectorphp/rector-src/pull/1610#issuecomment-1003574675, this change is not needed.
![image](https://user-images.githubusercontent.com/26657141/147854278-c2808ab0-5e96-44dc-a529-48c0d5459e54.png)